src/gui/optionsdlg.h

```diff
@@ -60,6 +60,8 @@ namespace Ui
 class OptionsDialog: public QDialog
 {
     Q_OBJECT
+    using ThisType = OptionsDialog;
```

Comment:
After some thoughts I would stick to `ThisType`. It reminds me that C++ does not make difference between user types and built-in ones, and realisation of that makes me happy every time :)